newIDE/app/src/UI/CompactTextField/UseClickDragAsControl.js

```diff
@@ -17,7 +17,11 @@ const useClickDragAsControl = ({ onChange, onGetInitialValue }: Props) => {
   const start = React.useCallback(
     (e: MouseEvent) => {
       const target = e.currentTarget;
-      if (target instanceof Element) {
+      if (
+        target instanceof Element &&
+        // requestPointerLock does not exist no mobile devices.
```

Comment:
so what happens on mobile, does this function gets not triggered?

Reply:
Indeed it does not work on mobile but the icons are too small for them to properly work I think, on a UX point of view. I'll check.

Reply:
Indeed it does not work on mobile but the icons are too small for them to properly work I think, on a UX point of view. I'll check.

Reply:
Note that this UX pattern (drag an icon to modify a field) is not something I think we've seen on mobile. So might be worth just disabling it entirely.